fitsmap/support/integerTranslate.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+// Author: Mingyu Li (lmytime [at] hotmail.com)
+// Date: 2024 Nov 11
+// Description:
+// ====================
+// This script ensures that the x and y values of `translate3d` transformations are rounded to integers
+// whenever the move or zoom action on the map ends.
+// This helps to prevent the appearance of "white sub-pixel image borders" that occur due to floating point rendering
+// inaccuracies, which is a common issue especially in Google Chrome.
+// Note that this script can lead to slight shifts in the map position after a move or zoom operation, but it is generally acceptable.
+// ====================
+
+// Function to round the x and y components of the `translate3d` CSS transformation to integer values.
+// This function is executed after the map ends a movement or zoom operation.
+const integerTranslateMapPane = function () {
```

Comment:
rather than rely on the global `map` variable it might be better to get the map object from the event, this would be robust to changes in the names of variables in `index.js` Change to something like:

```js
const integerTranslateMapPane = function (event) {
    // Obtain the map pane element, which contains the current translation transformation information.
    var mapPane = event.target.getPane('mapPane');
```